### ISSUE TITLE
Trabajo en grupo

### ISSUE THREAD

**Issue by giovannakbl** [giovannakbl/todo-list#1]
Hola! Vamos a trabajar en conjunto. Empezaré con mi tarea y avísote cuando puedas seguir con tuya. @Kaiisea

**Comment by Kaiisea** [comment]
ok, if it would be easier for you to communicate in english, I wouldn't have any problem. I'm waiting your update

**Comment by Kaiisea** [comment]
ok @giovannakbl , if it would be easier for you to communicate in english, I wouldn't have any problem. I'm waiting your update

**Comment by giovannakbl** [comment]
Ok! I am still working on it, sorry for the wait. I will let you know once it is ready @Kaiisea

**Comment by giovannakbl** [comment]
Hello @Kaiisea Could you check if you are able to access the changes?

**Comment by Kaiisea** [comment]
@giovannakbl Yes, I'm able to pulled it in my local. I will start with the first task: doing a project title

**Comment by giovannakbl** [comment]
Oh sorry. I understood that I was already supposed to do the first one. And I believe that we are supposed to make a mark on the things that we have done, and I forgot to do that in the to do list for the first task. @Kaiisea

**Comment by Kaiisea** [comment]
@giovannakbl Im not sure but have you already did the project title in README.md?

**Comment by giovannakbl** [comment]
Maybe you could do the getting started? Otherwise I can erase what I did and pull it again. What option do you believe would be better? @Kaiisea

**Comment by Kaiisea** [comment]
no worries @giovannakbl , we can distribute it differently. Should I did the next two tasks to compensate it?

**Comment by giovannakbl** [comment]
Sure! @Kaiisea

**Comment by Kaiisea** [comment]
one question @giovannakbl , in prerequisites i must add you examples linked with an url, do you know what url i should use?

**Comment by giovannakbl** [comment]
I believe that you can put whatever you prefer. Something like google.com. @Kaiisea

**Comment by Kaiisea** [comment]
Im having problems to push the information, i try to use my username and pass and terminal didnt recognize it @giovannakbl I will try to solve it

**Comment by giovannakbl** [comment]
No problem! No need to hurry. The problem is when the window to allow access to git hub opens? Not sure if I can help. @Kaiisea

**Comment by Kaiisea** [comment]
![Sin título](https://user-images.githubusercontent.com/99834510/154554068-075b93e1-a302-43c5-bcd0-261134fd2c40.png)
this is the problem I have when I push the information

**Comment by giovannakbl** [comment]
Have you tried using the token we generated yesterday as the password? I did the push via the buttons in VS Code. I was going to do the second one via terminal. Maybe the same problem will happen with me when I try that. @Kaiisea

**Comment by Kaiisea** [comment]
Sorry for my delay, my part is up now @giovannakbl I get it finally using the token as a password as you said. Thank you for the tip

**Comment by Kaiisea** [comment]
When you pull it, can you see part I did?

**Comment by giovannakbl** [comment]
Oh great that the token worked! It is not updating for me. Could you check if your commit appears for you in github? In the code area I can only see my commit for now. @Kaiisea

**Comment by Kaiisea** [comment]
Is it now up?

**Comment by giovannakbl** [comment]
Now it worked! Thanks! I will continue with the next task @Kaiisea

**Comment by giovannakbl** [comment]
I did the update. But I don't know what kind of element should be used to do the boxes around the Give the example and until finished texts. I believe I used the wrong one. But you can continue with the next one @Kaiisea

**Comment by Kaiisea** [comment]
deployment was too short, I have already did it. I added spaces between titles and texts as an extra cosmetic change @giovannakbl

**Comment by Kaiisea** [comment]
I confirmed the upload of deployment part @giovannakbl Its available to be downloaded

**Comment by giovannakbl** [comment]
@Kaiisea I just did a new commit but I realized I did something wrong. I will just try to correct it and commit it again

**Comment by Kaiisea** [comment]
no worries @giovannakbl

**Comment by giovannakbl** [comment]
@Kaiisea I did a new commit. You can proceed!

**Comment by Kaiisea** [comment]
I had an enormous problem with branches that I took a long time to understand and fix. Finally, I fixed all the bugs it caused. Authors finished @giovannakbl

**Comment by giovannakbl** [comment]
Is everything ok then? Did you create a branch on purpose or by accident? I don`t really know how to use branches jaja that is why I am asking. I will continue with the next one @Kaiisea

**Comment by Kaiisea** [comment]
by accident, and I didnt know how to eliminate it. After that, I was obligated to merge them in main one and that caused a lot of bugs.

**Comment by giovannakbl** [comment]
At least now you know how to merge branches! Advanced in class! I still need to learn that. I believe we are done with the task. If you could check please and let me know if it is ok, it would be great! @Kaiisea

**Comment by Kaiisea** [comment]
I forget to add bold to our usernames in Authors part. I have just done it. Exercise is finished, I think. Sorry for all the problems I had and thank you too much for your patience @giovannakbl

**Comment by giovannakbl** [comment]
It took me a long time to do it as well, and I got a lot of things wrong. No worries. I think we did a great job! I will just do a last correction, because I noticed I forgot to type a word in the text.  @Kaiisea

**Comment by giovannakbl** [comment]
Done! Thanks for the patience as well @Kaiisea
